src/app/admin/admin-reports/filtered-collections/filtered-collections.component.ts

```diff
@@ -0,0 +1,90 @@
+import { Component, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { NgbAccordion } from '@ng-bootstrap/ng-bootstrap';
+import { Observable } from 'rxjs';
+import { RestRequestMethod } from 'src/app/core/data/rest-request-method';
+import { DspaceRestService } from 'src/app/core/dspace-rest/dspace-rest.service';
+import { RawRestResponse } from 'src/app/core/dspace-rest/raw-rest-response.model';
+import { environment } from 'src/environments/environment';
+import { FiltersComponent } from '../filters-section/filters-section.component';
+import { FilteredCollections } from './filtered-collections.model';
+
+@Component({
+  selector: 'ds-report-filtered-collections',
+  templateUrl: './filtered-collections.component.html',
+  styleUrls: ['./filtered-collections.component.scss']
+})
+export class FilteredCollectionsComponent {
+
+  queryForm: FormGroup;
+  results: FilteredCollections = new FilteredCollections();
+  @ViewChild('acc') accordionComponent: NgbAccordion;
+
+  constructor(
+    private formBuilder: FormBuilder,
+    private restService: DspaceRestService) {}
+
+  ngOnInit() {
+    this.queryForm = this.formBuilder.group({
+      filters: FiltersComponent.formGroup(this.formBuilder)
+    });
+  }
+
+  filtersFormGroup(): FormGroup {
+    return this.queryForm.get('filters') as FormGroup;
+  }
+
+  getGroup(filterId: string): string {
+    return FiltersComponent.getGroup(filterId).id;
+  }
+
+  submit() {
+    this
+      .getFilteredCollections()
+      .subscribe(
+        response => {
+          this.results.deserialize(response.payload);
+          this.accordionComponent.expand('collections');
+        }
+      );
+  }
+
+  getFilteredCollections(): Observable<RawRestResponse> {
+    let params = this.toQueryString();
+    if (params.length > 0) {
+       params = `?${params}`;
+    }
+    let scheme = environment.rest.ssl ? 'https' : 'http';
+    let urlRestApp = `${scheme}://${environment.rest.host}:${environment.rest.port}${environment.rest.nameSpace}`;
+    return this.restService.request(RestRequestMethod.GET, `${urlRestApp}/api/contentreport/filteredcollections${params}`);
+  }
+
+  private toQueryString(): string {
+    let params = FiltersComponent.toQueryString(this.queryForm.value.filters);
+    return params;
+  }
+
+  /*
+  downloadCsv() {
```

Comment:
If you want to keep the commented out CSV related code here, then we should add a note as to why it's comment out.  Or, you can simply remove it, and we can add it back later once it is implemented.

Reply:
This code slipped through my eyes when I made my last corrections this weekend... But anyway, it's an opportunity to tell you about what I did so far.

I implemented a CSV save feature in the REST layer for the Filtered Collections report. It works, but I didn't include it (so far) in my PR because (1) I didn't have time to do the same with the Filtered Items report, and (2) I didn't succeed (yet) in implementing a download link/button in Angular.

Therefore, when I address your requests tomorrow morning, I will just remove this chunk of code, which anyway isn't working yet.

Reply:
Done.